packages/serve/src/parseArgs.ts

```diff
@@ -23,13 +23,8 @@ type ArgsType = {
  * @returns {Object} parsed webpack args and dev server args objects
  */
 export default function parseArgs(cli: WebpackCLIType, args: string[]): ArgsType {
-    let devServerFlags: object[];
-    try {
-        // eslint-disable-next-line node/no-extraneous-require
-        devServerFlags = require('webpack-dev-server/bin/cli-flags').devServer;
-    } catch (err) {
-        throw new Error(`You need to install 'webpack-dev-server' for running 'webpack serve'.\n${err}`);
-    }
+    // eslint-disable-next-line node/no-extraneous-require
+    const devServerFlags = require('webpack-dev-server/bin/cli-flags').devServer;
```

Comment:
Same logic goes here too, error can occur here as well if dev server changes files.

Reply:
Yes, you are right

Reply:
done